lib/Common/Memory/RecyclerPointers.h

```diff
@@ -489,21 +489,26 @@ template <class Policy>
 struct _QuickSortImpl
 {
     template<class T, class Comparer>
-    static void Sort(T* arr, size_t count, const Comparer& comparer, void* context)
+    static void Sort(T* arr, size_t count, const Comparer& comparer, void* context, size_t elementSize = sizeof(T))
     {
+#ifndef _WIN32
+        JsUtil::QuickSort<Policy, char, Comparer>::Sort((char*)arr, count, elementSize, comparer, context);
```

Comment:
Yes, I see that... I was asking to cleanup the JsUtil::QuickSort API so both calls are clean and consistent without casting/tweaking elementSize. That doesn't conflict with current behavior.